packages/revm/src/createRevm.js

```
@@ -0,0 +1,128 @@
+   */
+  const loadWasmModule = async () => {
+    try {
+      const module = await import("../pkg/tevm_revm.js");
+
+      const isNode =
+        typeof process !== "undefined" &&
+        process.versions != null &&
+        process.versions.node != null;
+
+      if (isNode) {
+        const fs = await import("fs/promises");
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)